app/helpers/events_helper.rb

```diff
@@ -1,26 +1,27 @@
 module EventsHelper
 
-  def user_is_attending_event
-    @event.attending?(current_user) if current_user
+  def user_is_attending_event(event = @event)
+    event.attending?(current_user) if current_user
   end
 
-  def attend_event_or_login(name, classes, additional_class_1, additional_class_2)
+  def attend_event_or_login(name, classes, additional_class_1,
+    additional_class_2)
     button_text = name || "Attend this event"
     css_class_attend = ""
     # css_class_attend =  "show" if user_is_attending_event
-    classes = classes || "modal-trigger btn waves-effect waves-light btn-large our-btn-green #{additional_class_1} #{additional_class_2}"
+    classes = classes || "modal-trigger btn waves-effect waves-light
+    btn-large our-btn-green #{additional_class_1} #{additional_class_2}"
 
     if current_user
-      link_to(button_text, '#purchase_ticket_modal', class: classes, data: {id: 'unattend', target: 'purchase_ticket_modal'}, id: 'attend')
+      link_to(button_text, '#purchase_ticket_modal', class: classes,
+              data: {id: 'unattend', target: 'purchase_ticket_modal'},
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.